### ISSUE TITLE
VSCode Insiders: Integrated terminal has a LANG=en_US.UTF-8 env variable declared

### ISSUE THREAD

**Issue by Nazeehe** [Microsoft/vscode#35550]
- VSCode Version: Insiders 1.17.0
- OS Version: Windows 10

Steps to Reproduce:

1. Set up a Windows CMD integrated terminal window
2. Launch it and inspect environment variables
3. Notice the LANG env variable? Where did that come from?

This env variable is messing with my build at work (Microsoft, Razzle). 

Reproduces without extensions: Yes

More info: Found the issue in the code:
d:\vscode-master\src\vs\workbench\parts\terminal\electron-browser\terminalInstance.ts
public static createTerminalEnv
env['LANG'] = TerminalInstance._getLangEnvVariable(locale);

More info: 
This is the issue that introduced this as a fix: https://github.com/Microsoft/vscode/issues/14586

Is there a way to not make it set those?

**Comment by Tyriar** [comment]
Is the variable causing particular issues? This is an environment variable used on Linux/macOS but should also be useful for WSL bash sessions.

**Comment by Nazeehe** [comment]
Hey Tyriar,

Yeah, I got a build error building Windows due to this env variable being set. I am using VSCode with my Razzle window at Microsoft. Once I unset it, the build succeeded. 

It's an interesting thing though. I can see that you guys are setting a bunch of env variables in there for the terminal. You also provide a way for the user to add more. But no way to unset them? 

Any how, I worked around it by having my startup script in razzle unset it. I am sure you guys will find a great way to fix it :)
